packages/mds-stream/index.ts

```diff
@@ -54,9 +53,7 @@ const getBinding = () => {
 
 const getNats = () => {
   if (!nats) {
-    nats = stan.connect(env.STAN_CLUSTER || 'stan', `mds-agency-${uuid()}`, {
-      url: `nats://${env.NATS}:4222`,
-      userCreds: env.STAN_CREDS,
+    nats = NATS.connect(`nats://${env.NATS}:4222`, {
```

Comment:
Shouldn't the entire URL (or `NATS_HOST`) come from `process.env` (perhaps using getEnvVar?

Reply:
In theory, that isn't how we've structured the host to date, though. Would require some changes for our helm charts that I don't think we should implement until https://github.com/lacuna-tech/mds-core/pull/185 lands

Reply:
I think a `NATS_HOST` that defaults to `localhost:4222` would be consistent with how we handle other network services.

Reply:
Refactored to use the new getEnvVar pattern, but haven't switched to `NATS_HOST` which would include protocol prefix and port suffix yet. Will wait on that until #185 lands.